app/src/components/LayerControlBody.vue

```diff
@@ -610,7 +710,43 @@ export default {
         name: `flip on axis ${axis}`
       })
       this.flipAxis({ axis })
+    }, 
+    setResolution: function () {
+      const isotropic = +this.selectedResolutionX === +this.selectedResolutionY && +this.selectedResolutionY === +this.selectedResolutionZ
+
+      if (isotropic) {
+        this.setIsotropicScale(+this.selectedResolutionX/this.selectedIncomingVolumeResolutionX)
+      } else {
+          this.isotropic = false
```

Comment:
ditto above. 

`this.isotropic` should be a state that is *exclusively* controlled by the user.  the application should not be able to influence it.